src/components/web3-browser/web3-browser-container.tsx

```diff
@@ -0,0 +1,72 @@
+// @refresh reset
+import React, {useCallback} from 'react';
+
+import {useFocusEffect} from '@react-navigation/native';
+import {Keyboard, KeyboardEventListener, View} from 'react-native';
+import {KeyboardAwareScrollView} from 'react-native-keyboard-aware-scroll-view';
+
+import {createTheme} from '@app/helpers';
+import {IS_ANDROID, IS_IOS} from '@app/variables/common';
+
+import {Web3BrowserProps} from './web3-browser';
+
+export function WebViewContainer({
+  children,
+  webviewRef,
+}: {
+  webviewRef: Web3BrowserProps['webviewRef'];
+  children: any;
+}) {
+  if (IS_IOS) {
+    return (
+      <KeyboardAwareScrollView
+        bounces={false}
+        contentContainerStyle={styles.webviewContainer}>
+        {children}
+      </KeyboardAwareScrollView>
+    );
+  }
+
+  if (IS_ANDROID) {
+    // eslint-disable-next-line react-hooks/rules-of-hooks
+    useFocusEffect(
```

Comment:
It should crash... Because you have unknown hook count. Could you please add `if (IS_ANDROID) return;` in `useCallback` to make sure you always have `useFocusEffect` hook?

Or you have tested this already and it works fine for both platforms?

Reply:
I want avoid unnecessary useFocusEffect on iOS.  `if (IS_ANDROID)` work correctly  and code block execute only for android without crash ` unknown hook count`